ros/scripts/joint_states_to_float

```diff
@@ -18,7 +18,7 @@ class JointStatesToFloat:
         # init previous angle list
         self.previous_angles = [0] * len(self.joint_state_msg.position)
         # create one publisher
-        self.pub = rospy.Publisher('/position_controller/command', Float64MultiArray, queue_size=1)
+        self.pub = rospy.Publisher(namespace + 'position_controller/command', Float64MultiArray, queue_size=1)
```

Comment:
is this safe? Maybe `rospy.get_namespace()` will always return a namespace ending with `/`, but another caller might not. I would suggest the following, might be overkill though:

```suggestion
        self.pub = rospy.Publisher(os.path.join(namespace, 'position_controller/command'), Float64MultiArray, queue_size=1)
```

Reply:
Well that's why I don't call it with `rospy.get_namespace()` but with `robot.get_namespace()` and that should be okay because in the initialization of the robot, this happens:
```python
allowed = re.compile("[a-zA-Z0-9_]*$")
if not allowed.match(name):
    rospy.logerr(
        "[PyBulletRobot::init] Invalid name '{}' for a PyBulletRobot. Allowed characters are [a-zA-Z0-9_]. " +
        "Exiting now.".format(name))
    return
self._namespace = "/" + name + "/"
```

Reply:
No, that should work just fine!